src/euphorie/client/docx/views.py

```diff
@@ -308,7 +308,10 @@ def _filename(self):
 
 
 class ActionPlanShortDocxView(ActionPlanDocxView):
-    _compiler = DocxCompilerShort
+    @property
+    @memoize
+    def _compiler(self):
+        return DocxCompilerShort
```

Comment:
This customization should not be needed, anyway good job.

Please double check if this can removed. In that case clean up and merge :)

Reply:
In the `ActionPlanShortDocxView` I always want to use the `DocxCompilerShort` which is never used in `ActionPlanDocxView`. This will not be the case if I remove this override. Or am I misunderstanding something?